hack/jenkins/e2e.sh

```diff
@@ -98,6 +191,7 @@ export KUBE_SKIP_CONFIRMATIONS=y
 export E2E_UP="${E2E_UP:-true}"
 export E2E_TEST="${E2E_TEST:-true}"
 export E2E_DOWN="${E2E_DOWN:-true}"
+export GINKGO_PARALLEL=${GINKGO_PARALLEL:-}
```

Comment:
... worth comment on how this is used? (I assume this is read by Ginkgo itself?)